CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Moved `CornerStyle` out of the `Box` namespace, and is now a root type in `BlueprintUICommonControls`. `Box.CornerStyle` is still available as a typealias.
+- `BlueprintView` will now pass through touches to views lower in the view hierarchy if `passThroughTouches` is true.
```

Comment:
this is a non-breaking addition, right?

Reply:
Ahh yep, good catch. I'd originally just made this the only behavior, but figured that was a bad idea.

- [ ] Fix Changelog